docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
 
 ## Data
 
+### Key Tables
+
+| **Name**<br>*Key(s)*| **Description** |
+| :------------- |  :-------------- |
+| `organizations`<br><br>*External*: `Name` | Records are legal organizations, including companies, governmental bodies, or non-profits. <br><br>Table includes information on organizational properties (i.e. locale, type) as well as summarizations of its various relationships (e.g. `services` for a transit provider, or `products` for a vendor). <br> <br>An organization MAY:<br><ul><li>*manage* one more more `services`</li><li>*operate* one more `services`</li><li>*own* one more `contracts`</li><li>*hold* one more `contracts`</li><li>*sell* one more `products`</li><li>*consume* one more `gtfs datasets`</li><li>*produce* one more `gtfs datasets`</li></ul>
```

Comment:
What do these `External:`, `Internal:` things mean? Please document their meanings.

Reply:
Yeah - that was confusing so thanks for flagging. After re-reading it I decided to focus on documenting the Airtable concept of Primary Field and noting them rather than worry about documenting if/where the RECORD ID (the real primary key, mostly hidden in Airtable) is exposed as a field. 

See if the updates in https://github.com/cal-itp/data-infra/pull/802/commits/08a0e42dc20a7b86689f9cbde1cb33efe88e761f make this more straightforward.